content/engine/release-notes/25.0.md

```diff
@@ -28,6 +28,16 @@ For a full list of pull requests and changes in this release, refer to the relev
 - [docker/cli, 25.0.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A25.0.0)
 - [moby/moby, 25.0.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A25.0.0)
 
+> **Note**
+>
+> In earlier versions of Docker Engine, recursive mounts (submounts) would
+> always be mounted as writable, even when specifying a read-only mount. This
+> behavior has changed in v25.0.0, for hosts running on kernel version 5.12 or
+> later. Now, all recursive mounts are read-only by default.
+>
+> For more information, see
+> [Recursive mounts](../../storage/bind-mounts.md#recursive-mounts).
```

Comment:
Would be also nice to include that to have the equivalent of the old behavior (`-v src:dst:ro`) is 
`--mount type=bind,src=SRC,dst=DST,readonly,bind-recursive=writable`.